 --- src/texmacs-mygit/src/System/Boot/init_texmacs.cpp.orig	2016-07-10 15:32:15.871756600 +0200
+++ src/texmacs-mygit/src/System/Boot/init_texmacs.cpp	2016-07-10 15:30:18.135217200 +0200
@@ -20,6 +20,7 @@
 #include <unistd.h>
 #ifdef __MINGW32__
 #include <time.h>
+#include <direct.h>
 #endif
 
 tree texmacs_settings = tuple ();
